.travis.yml

```diff
@@ -6,10 +6,14 @@ node_js:
   - 'node'
   - '7'
   - '6'
+  - '4'
 before_script:
   - npm prune
 script:
-  - npm run validate
+  # detect if the current build's node version is < 6
+  # if yes, only test coverage
+  # if not, lint and test coverage (full validation)
+  - if [[ `node -v | sed 's/[^0-9\.]//g'` < 6 ]]; then npm run cover; else npm run validate; fi
```

Comment:
Thanks for the hint. I planned on trying that, might be a good opportunity to do so! :+1: